vip-jetpack/connection-pilot/class-jetpack-connection-controls.php

```diff
@@ -288,6 +302,8 @@ private static function maybe_create_user() {
 	 * This helps prevent cache issues for times where the database was directly updated.
 	 */
 	private static function refresh_options_cache() {
+		wp_cache_flush_runtime();
```

Comment:
Small addition just to help ensure the request doesn't carry over some other stale object cache values that may have been updated via the roundtrips with Jetpack.